docs/lib/datastore/fragments/android/getting-started/20_installLib.md

```diff
@@ -0,0 +1,21 @@
+<!--TODO Update AWSmobile Client -> Auth -->
+Add the following dependencies to your **app** build.gradle file and click "Sync Now" when asked:
+
+```groovy
+dependencies {
+    implementation 'com.amplifyframework:core:0.10.0'
+    implementation 'com.amplifyframework:aws-storage-s3:0.10.0'
+    implementation 'com.amazonaws:aws-android-sdk-mobile-client:2.16.+'
```

Comment:
do you mind if i just update these on this branch?
```
implementation 'com.amplifyframework:aws-datastore:1.0.0'
implementation 'com.amplifyframework:aws-auth-cognito:1.0.0'
```

Reply:
don't worry, all the code snippets for all platforms will be updated